### DISCUSSION TITLE
Enable Sponsors button?

### DISCUSSION THREAD

**Discussion by tiesont** [hey-red/ImageSharp.AVCodecFormats#6] [General]
I have no idea what's involved in doing so, but have you thought about enabling [Sponsors](https://github.com/sponsors) for your project?

I'm currently building something that already leverages ImageSharp for image processing, and FFMpegCore (a .NET Core wrapper around ffmpeg) for video processing. From a cursory glance at this project, I think I could simplify what I'm doing by using this either in place of, or in addition to, my current solution. If I do, I'd like to make sure we kick something back your way, and Sponsors is a simple way for me to do so.

**Comment by hey-red** [comment]
Hello! Github sponsors is not available in my country, so I just add link to opencollective.
